.github/workflows/fpga-subsystem.yml

```diff
@@ -352,3 +352,24 @@ jobs:
           path: |
             /tmp/junit.xml
             /tmp/nextest-list.json
+
+  print_test_results:
+    runs-on: ubuntu-22.04
+    needs: [test_artifacts_subsystem]
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 #v5
+        with:
+          ref: ${{ inputs.branch }}
+          submodules: recursive
+
+      - name: 'Download test results'
+        uses: actions/download-artifact@v4
+        with:
+          name: caliptra-test-results${{ inputs.artifact-suffix }}
+          path: /tmp/junit.xml
+
+      - name: Print results
+        run: |
+          echo "## Test Results\n" >> $GITHUB_STEP_SUMMARY
```

Comment:
TODO: Follow up, remove this \n. 

Skipping now to avoid generating more CI jobs.